components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -634,7 +680,13 @@ public OAuthAppDO getAppInformationByAppName(String appName) throws
                                     (4));
                             oauthApp.setOauthConsumerKey(preprocessedClientId);
                             oauthApp.setOauthVersion(rSet.getString(5));
+
```

Comment:
redundant blank line

Reply:
Added it to improve readability by separating the callback URL resolving logic.